core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -759,6 +759,14 @@ public String toString() {
           .setDefaultValue(19998)
           .setDescription("The port that Alluxio master node runs on.")
           .build();
+  public static final PropertyKey MASTER_SERVING_THREAD_TIMEOUT =
```

Comment:
@aaudiber  are we supposed to add new properties in a patch release?
if we do, the configuration webpage will be tricky since it is per minor release.
Alternatively, for 1.7, shall we simply hard code the value and introduce this as a new property in master?

Reply:
saw the hidden field.
SGTM